packages/components/src/internal/components/domainproperties/__snapshots__/DomainForm.spec.tsx.snap

```diff
@@ -381,6 +383,31 @@ exports[`DomainForm domain form add, expand, and delete field 1`] = `
                               </div>
                             </div>
                           </ActionButton>
+                          <ActionButton
```

Comment:
Does this change mean a default value has been swapped?  Probably that's intentional, but just wanted to be sure.

Reply:
Yes, the switch from "showImportExport" to "hideImportExport" flipped the default. I could have reset the old value in this test case, but decided not to to reflect the new default value.